pkg/operator/target_config_reconciler_v311_00.go

```diff
@@ -127,19 +130,23 @@ func manageKubeApiserverPublicConfigMap_v311_00_to_latest(client coreclientv1.Co
 	return resourceapply.ApplyConfigMap(client, configMap)
 }
 
-func managePod_v311_00_to_latest(client coreclientv1.ConfigMapsGetter, options *v1alpha1.KubeApiserverOperatorConfig) (*corev1.ConfigMap, bool, error) {
+func managePod_v311_00_to_latest(client coreclientv1.ConfigMapsGetter, operatorConfig *v1alpha1.KubeApiserverOperatorConfig) (*corev1.ConfigMap, bool, error) {
 	required := resourceread.ReadPodV1OrDie(v311_00_assets.MustAsset("v3.11.0/kube-apiserver/pod.yaml"))
-	switch corev1.PullPolicy(options.Spec.ImagePullPolicy) {
+	switch corev1.PullPolicy(operatorConfig.Spec.ImagePullPolicy) {
 	case corev1.PullAlways, corev1.PullIfNotPresent, corev1.PullNever:
-		required.Spec.Containers[0].ImagePullPolicy = corev1.PullPolicy(options.Spec.ImagePullPolicy)
+		required.Spec.Containers[0].ImagePullPolicy = corev1.PullPolicy(operatorConfig.Spec.ImagePullPolicy)
 	case "":
 	default:
-		return nil, false, fmt.Errorf("invalid imagePullPolicy specified: %v", options.Spec.ImagePullPolicy)
+		return nil, false, fmt.Errorf("invalid imagePullPolicy specified: %v", operatorConfig.Spec.ImagePullPolicy)
 	}
-	required.Spec.Containers[0].Image = options.Spec.ImagePullSpec
-	required.Spec.Containers[0].Args = append(required.Spec.Containers[0].Args, fmt.Sprintf("-v=%d", options.Spec.Logging.Level))
+	required.Spec.Containers[0].Image = operatorConfig.Spec.ImagePullSpec
+	// TODO remove
```

Comment:
why hardcoded?

Reply:
> why hardcoded?

The installer pod has the wrong image and I need to look up how use the downward API